PROJ-15 sprint 1 goal (#9)
SW805F20/report: sections/sprint-1/sprint-goal.tex

\section{Sprint goal}
The goal for this sprint is to explore the project idea and look for possible solutions to some of the problems that the project idea introduces.

> In this sprint we want to learn more about how we can visualize the game for the users so they can see them selves in the game as well as the playing field.

Reviewer: ```suggestion
In this sprint, we want to learn more about how we can visualize the game for the users so they can see themselves in the game as well as the playing field.
```

> We want to look at how we can get the playing field's as well as the players' positions in an accurate way with fast updates.

Reviewer: ```suggestion
We want to look at how we can get the playing field as well as the players' positions in an accurate way with fast updates.
```

> As this is the first sprint of the project, the main focus is to gain more knowlegde about the different aspects of the project, and to gain a shared vision amongst the group members of how the game is going to work.

Reviewer: ```suggestion
As this is the first sprint of the project, the main focus is to gain more knowledge about the different aspects of the project and to gain a shared vision amongst the group members of how the game is going to work.
```

> The shared vision will be achieved by making prototypes of the game as well as architectual diagrams of how different components in the game will work together.

Reviewer: ```suggestion
The shared vision will be achieved by making prototypes of the game as well as architectural diagrams of how different components in the game will work together.
```
